types/moddable/tslint.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
```

Comment:
It looks like you're disabling a lot of the linting rules we have on by default. Ideally, they should all be enabled, unless there is a good reason to disable a rule. Is it the case here? Do you think you could refactor these definitions so that you wouldn't need to disable all of those rules?

Reply:
Hi @gabritto 
I have reduced the exception rules disable statement to the following:
```json
  "rules": {
    "no-single-declare-module": false,
    "no-unnecessary-class": false,
    "unified-signatures": false,
    "no-var-keyword": false
  }
```

Because this original package actually a completely separated JavaScript runtime environment that is implemented in C and not sharing any binary with NodeJS or Browser interpreter, the rules disabled in Node Types definition is suitable to be compared here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/tslint.json
```json
  "rules": {
    "ban-types": false,
    "no-empty-interface": false,
    "no-outside-dependencies": false,
    "no-single-declare-module": false,
    "no-unnecessary-class": false,
    "unified-signatures": false
  }
```

Because the API implemented in C binaries and exposed to JavaScript interpreter is with no chance to do any definition improvement, the classes signatures have to be kept as same as the interface exposed from C libraries.

Regarding the `var` keyword, this is possible to be removed in the future while I initially fixed the same problem in original package's typing files.